src/H5Sparse.jl

```diff
@@ -52,12 +52,13 @@ Matrix(A)           # Matrix
 ```
 
 """
-struct H5SparseMatrixCSC{Tv, Ti<:Integer} <: SparseArrays.AbstractSparseMatrixCSC{Tv, Ti}
-    fid::HDF5.H5DataStore          # Backing HDF5 file
+
+struct H5SparseMatrixCSC{Tv, Ti<:Integer, Td<:HDF5.H5DataStore} <: SparseArrays.AbstractSparseMatrixCSC{Tv, Ti}
+    fid::HDF5.H5DataStore   # Backing HDF5 file
```

Comment:
Change this line to `fid::Td                 # Backing HDF5 store`

Reply:
done!